src/index.ts

```diff
@@ -3,6 +3,7 @@ export {default as getLogger, Logger} from './logging';
 export {default as Driver} from './driver';
 export {
     declareType,
+    StructFields,
```

Comment:
Commit suggestion

feat: add more methods to type helper classes to support more non-primitive types 
-> 
feat: add more methods to type helper classes

Then elaborate shortly, which methods you are adding